docs/tutorial/maps.rst

```diff
@@ -181,748 +168,387 @@ Some basic statistical functions are built into Map objects:
-Matplotlib handles this colormapping process through the `~matplotlib.colors` module.
-First, the data array is mapped onto the range 0-1 using an instance of `~matplotlib.colors.Normalize` or a subclass.
-Then, the data is mapped to a color using a `~matplotlib.colors.Colormap`.
-
-**sunpy** provides colormaps for each mission as defined by the mission teams.
-The Map object chooses the appropriate colormap for you when it is created as long as it recognizes the instrument.
-To see what colormaps are available:
-
-.. code-block:: python
-
-    >>> import sunpy.visualization.colormaps as cm
-    >>> cm.cmlist.keys()
-    dict_keys(['goes-rsuvi94', 'goes-rsuvi131', 'goes-rsuvi171', 'goes-rsuvi195',
-    'goes-rsuvi284', 'goes-rsuvi304', 'sdoaia94', 'sdoaia131', 'sdoaia171',
-    ...
-
-The **sunpy** colormaps are registered with Matplotlib so you can grab them like you would any other colormap:
-
-.. code-block:: python
+    from astropy.coordinates import SkyCoord
+    import astropy.units as u
+    my_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
 
-    >>> import matplotlib.pyplot as plt
-    >>> import sunpy.visualization.colormaps
-    >>> cmap = plt.get_cmap('sdoaia171')
+In the :ref:`creating-maps` section, you learned how to generate a quicklook summary of a Map.
+However, the Map object also has a :meth:`~sunpy.map.GenericMap.plot` method that allows for more fine-grained control over how the Map is visualized and is especially useful for generating publication-quality plots.
+In this section of the tutorial, you will learn how to build up an increasingly detailed visualization of a Map, including adjusting the colormap and normalization and and overlaying coordinates and contours.
 
-See `~sunpy.visualization.colormaps` for a plot of all available colormaps.
+Basic Plotting
+--------------
 
-If you want to override the built-in colormap, consider the following example which plots an AIA map using an EIT colormap.
+First, let's create a basic plot of our Map, including a colorbar,
 
 .. plot::
     :include-source:
+    :context: close-figs
 
-    import sunpy.map
-    import sunpy.data.sample
     import matplotlib.pyplot as plt
-
-    aia_map = sunpy.map.Map(sunpy.data.sample.AIA_171_IMAGE)
-    cmap = plt.get_cmap('sohoeit171')
-
     fig = plt.figure()
-    aia_map.plot(cmap=cmap)
+    ax = fig.add_subplot(projection=my_map)
+    my_map.plot(axes=ax)
     plt.colorbar()
     plt.show()
 
-You can also change the colormap for the Map itself:
-
-.. code-block:: python
+.. note::
 
-    >>> smap.plot_settings['cmap'] = plt.get_cmap('sohoeit171')  # doctest: +SKIP
+    We imported `matplotlib.pyplot` in order to create the figure and the axis we plotted on our map onto.
+    Under the hood, sunpy uses of `matplotlib` to visualize the image meaning that plots built with sunpy can be further customized using `matplotlib`.
+    **However, for the purposes of this tutorial, you do not need to be familiar with Matplotlib.**
```

Comment:
I think WCSAxes should be mentioned here somewhere no? just so that people know that its not _exactly_ the same as matplotlib by itself

Reply:
Fair point. I don't think the main body of the tutorial is a good place, but this call out box is probably a good place to link this information.